npm-search/assets/css/base.css

```diff
@@ -5,6 +5,11 @@ body {
   margin: 0; 
 } 
 
+h1 {
+  -webkit-margin-before: 0.3em;
```

Comment:
We should consider removing the wrapping altogether. Also, this change is webkit specific and likely does not work on IE.